src/Glpi/Form/Destination/CommonITILField/EntityField.php

```diff
@@ -182,4 +184,48 @@ public function getCategory(): Category
     {
         return Category::PROPERTIES;
     }
+
+    #[Override]
+    public function convertFieldConfig(FormMigration $migration, Form $form, array $rawData): JsonFieldInterface
+    {
+        /**
+         * FormCreator implements many strategies to determine the entity.
+         * Some strategies are not 100% supported by the new form system:
+         * - 2. Default requester user's entity
+         *  This strategy take the entity of the first requester user and fallback to the form filler entity.
+         *
+         * Strategies that are supported:
+         * - 3. First dynamic requester user's entity (alphabetical) -- Must be implemented
+         * - 4. Last dynamic requester user's entity (alphabetical) -- Must be implemented
+         * - 6. Default entity of the validator
+         * - 8. Default entity of a user type question answer
+         */
+
+        switch ($rawData['destination_entity']) {
+            case 1:
+            case 2:
```

Comment:
You should add a comment to shortly explain what the number is corresponding to, e.g. `case 1: // PluginFormcreatorAbstractTarget::DESTINATION_ENTITY_CURRENT`.

Any developer that would have to investigate on a bug would be happy to have some hint about what this value is supposed to do in FormCreator.

This is a global remark, even for tests, that are pretty hard to validate right now.